### ISSUE TITLE
NER模型预测为空

### ISSUE THREAD

**Issue by ume-technology** [zjunlp/DeepKE#160]
**Describe the question**
> 我标注了一些数据，但是无论怎样，执行predict后，预测的实体都是空。甚至尝试预测了train data中的一条数据，结果同样为空。
我使用是 standard NER模型。
使用的预训练模型是在配置文件中是这样的，配置在文件夹的这里：\conf\hydra\model\bert.yaml，在这个文件中配置了我本地的pre model

我不清楚是哪里出了问题。

**Environment (please complete the following information):**
 - OS: [e.g. mac / window]
 - Python Version [e.g. 3.6]

**Screenshots**
> If applicable, add screenshots to help explain your problem.

**Additional context**
> Add any other context about the problem here.

**Comment by zxlzr** [comment]
请问您标注了多少数据，模型训练需要大约数千条样本。

**Comment by ume-technology** [comment]
可能是我的训练集太少了，几百条的量，但是为什么测试train data 也是空呢？

**Comment by zxlzr** [comment]
如果train 也是空的话建议您打印loss看一下，是否学习率等超参数设置的不合理，loss 如果一直下降的话多跑几个epoch 是可以对train 预测出结果的

**Comment by ume-technology** [comment]
我再查看一下代码吧。因为是我一个人标注，我之后会尝试一些弱标注的方案。这批数据集我在wandb上查看loss，发现有飙升的情况。因此已经尝试过设定新的epochs和学习率，但是traindata没有预测结果的情况依然没有改变。另外我认为已经是traindata了，和学习率已经没关系了，但是我没有找到问题的原因在哪里。总之感谢您的帮助，有不懂的再来请教。感谢。

**Comment by ume-technology** [comment]
另外请问，学习率是配置文件中的哪个参数，我怕我是不是该错位置了。

**Comment by zxlzr** [comment]
learning_rate: 1e-3

**Comment by qinglongheu** [comment]
> **Describe the question**
> 
> > 我标注了一些数据，但是无论怎样，执行predict后，预测的实体都是空。甚至尝试预测了train data中的一条数据，结果同样为空。
> > 我使用是 standard NER模型。
> > 使用的预训练模型是在配置文件中是这样的，配置在文件夹的这里：\conf\hydra\model\bert.yaml，在这个文件中配置了我本地的pre model
> 
> 我不清楚是哪里出了问题。
> 
> **Environment (please complete the following information):**
> 
> * OS: [e.g. mac / window]
> * Python Version [e.g. 3.6]
> 
> **Screenshots**
> 
> > If applicable, add screenshots to help explain your problem.
> 
> **Additional context**
> 
> > Add any other context about the problem here.

这个可能是应为你模型预测出的结果都是‘o’即没有识别出per, org等自己设计的标签， 建议换一下predict的文本数据。

**Comment by ume-technology** [comment]
收到，感谢您的帮助。我最近又会涉及到NER的任务。会再尝试检查问题。

qinglongheu ***@***.***> 于2022年12月13日周二 17:24写道：
